Source/Foundation/Data.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Data.swift
+//  Rex
+//
+//  Created by Ilya Laryionau on 10/05/15.
+//  Copyright (c) 2015 Neil Pankey. All rights reserved.
+//
+
+import ReactiveCocoa
+
+extension Data {
+    /// Read the data at the URL, sending the result or an error.
+    public static func rex_dataWithContentsOfURL(_ url: URL, options: Data.ReadingOptions = []) -> SignalProducer<Data, NSError> {
```

Comment:
I think this should be renamed to align with the signature of the new `Data` type.

```swift
public static func rex_data(contentsOf url: URL, options: Data.ReadingOptions = []) -> SignalProducer<Data, NSError>
```